cms/djangoapps/import_from_modulestore/models.py

```diff
@@ -0,0 +1,144 @@
+"""
+Models for the course to library import app.
+"""
+
+import uuid as uuid_tools
+from typing import Optional
+
+from django.contrib.auth import get_user_model
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from model_utils.models import TimeStampedModel
+from opaque_keys.edx.keys import UsageKey
+from opaque_keys.edx.django.models import (
+    LearningContextKeyField,
+    UsageKeyField,
+)
+from openedx_learning.api.authoring_models import LearningPackage, PublishableEntity
+
+from .data import ImportStatus
+
+User = get_user_model()
+
+
+class Import(TimeStampedModel):
+    """
+    Represents the action of a user importing a modulestore-based course or legacy
+    library into a learning-core based learning package (today, that is always a content library).
+    """
+
+    uuid = models.UUIDField(default=uuid_tools.uuid4, editable=False, unique=True)
+    status = models.CharField(max_length=100, choices=ImportStatus.choices, default=ImportStatus.PENDING, db_index=True)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+
+    source_key = LearningContextKeyField(help_text=_('The modulestore course'), max_length=255, db_index=True)
+    target = models.ForeignKey(LearningPackage, models.SET_NULL, null=True)
```

Comment:
```suggestion
    target_change = models.ForeignKey(DraftChangeLog, models.SET_NULL, null=True)
```
Similar to my previous comment: now that this model has merged into openedx_learning, we can use it here, which gives us better connectivity between the Import events and the new draft change log.

If you feel that it is right, feel free to add a `@property` to this model called `target` which simply returns `self.target_change.learning_package`.

Reply:
Changed.